mmv1/products/filestore/terraform.yaml

```diff
@@ -53,6 +53,22 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       constants: templates/terraform/constants/filestore.erb
       pre_create: templates/terraform/pre_create/filestore_instance.go.erb
+  Snapshot: !ruby/object:Overrides::Terraform::ResourceOverride
+    mutex: "filestore/{{project}}"
+    error_retry_predicates: ["isNotFilestoreQuotaError"]
```

Comment:
I'm assuming that snapshots will be subject to the same problem as filestore instances: if the quota is reached, there is no sense in retrying.